db/migrate/20240313132158_create_countries.rb

```diff
@@ -0,0 +1,9 @@
+class CreateCountries < ActiveRecord::Migration[7.1]
+  def change
+    create_table :countries do |t|
+      t.string :name
```

Comment:
Since there's a uniqueness validation in the model class, you'll want a constraint on the database level because otherwise, depending on the database configuration, you can still end up with two countries that have the same name (there's a warning about this on the [Rails guide](https://guides.rubyonrails.org/active_record_validations.html#uniqueness)).
 
```suggestion
      t.string :name, index: { unique: true }
```